README.md

```
@@ -10,315 +10,366 @@
-
-### 1. Software dependency - Postgres server
-
-- Download and start [postgres server](https://www.postgresql.org/download/)
-
-For Postgres, for the development server, we default to using postgres/postgres/postgres as the username/password/machine name - this is configurable.
-
-We tested and developed against the following versions:
+## Table of Contents
+
+* [Pre-requisites](#1-pre-requisites)
+   * [Software Dependencies](#software-dependencies)
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 3

(MD007, ul-indent)